request-attrs.go

```diff
@@ -28,23 +28,23 @@ func (r *Request) Pflags() FileOpenFlags {
 	return newFileOpenFlags(r.Flags)
 }
 
-// Flags that indicate whether SFTP file attributes were passed. When a flag is
+// FileAttrFlags that indicate whether SFTP file attributes were passed. When a flag is
 // true the corresponding attribute should be available from the FileStat
 // object returned by Attributes method. Used with SetStat.
 type FileAttrFlags struct {
-	Size, UidGid, Permissions, Acmodtime bool
+	Size, UIDGid, Permissions, Acmodtime bool
```

Comment:
Public field in public struct. This is going to have to stay for compatibility reasons.

Reply:
ok, this lint issue remains